### DISCUSSION TITLE
FitBit Versa 3 help/Advice needed!!

### DISCUSSION THREAD

**Discussion by smmdolfin424** [community/community#69677] [Actions]
My Versa 3 is trash, just like Fitbits customer service. That being said, Humana gives me perks or I would have switched to a fossil or a Garmin, for the almost 20 years I've been a loyal FitBit customer. This Versa 3 is the 2nd replacement in under a year. It's not tracking my sleep right, activities, nothing. There's numbers but I KNOW FOR FACTS that they are NOT accurate. I'm not getting the points for Humana, and I have insomnia, so the sleep tracking and recording IS OF EXTREME importance to me.  I need to know if you wonderful tech nerds, have some magic codes, or advice of what I can do on here, to make this worthless Versa 3,  do work son! 🤣😢

**Comment by moayad-star** [comment]
It's time to buy new

**Reply by smmdolfin424** [reply]
My insurance, Humana pays for it, and they only cover one a year, this is my 2nd. Every single fitbit I've used over the course of 20 ish years, has been crap. If I was paying out of pocket, I'd have been doing Garmin, moto, or Fossil, believe me! Lol

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
